addons/website/views/snippets.xml

```diff
@@ -900,7 +900,7 @@
                     <t t-snippet="website.s_cover" t-thumbnail="/website/static/src/img/snippets_thumbs/s_cover.png"/>
                     <t t-snippet="website.s_text_image" t-thumbnail="/website/static/src/img/snippets_thumbs/s_text_image.png"/>
                     <t t-snippet="website.s_image_text" t-thumbnail="/website/static/src/img/snippets_thumbs/s_image_text.png"/>
-                    <t t-snippet="website.s_title" t-thumbnail="/website/static/src/img/snippets_thumbs/s_title.png"/>
+                    <t t-snippet="website.s_title" t-thumbnail="/website/static/src/img/snippets_thumbs/s_title.png" t-forbid-sanitize="true"/>
```

Comment:
Actually there are sanitized snippets in 13.0 :wink: Forms cannot be added in 13.0 product descriptions for example :wink: We changed that for 14.0, considering forms safe for those https://github.com/odoo/odoo/commit/388c222c6c4bb7e2fe3e67009b248359ae0fd3db

I guess we could technically need your new `t-forbid-sanitize` to not be a boolean but like "all, accepts-forms, etc"... But let's forget about that for the moment I guess. In any case, in 13.0, you should just add your t-forbid-sanitize instruction on the form snippet :+1:

Reply:
Removed `s_title` and `s_hr` wip
Added `t-forbid-sanitize` on form builder and both product search blocks.